Dockerfile

```diff
@@ -1,12 +1,12 @@
 # build stage
-FROM golang:latest AS build-env
-RUN go get github.com/golang/dep/cmd/dep 
+FROM golang:1.11-alpine AS build-env
```

Comment:
Is there a way to not tie this to 1.11?

Reply:
github shows a linter error when using latest. Is there any concerns not using a dedicated tag?

Reply:
Let's use `golang:alpine`

Reply:
oh, just realized the alpine 1.11 has missing packages. Updated.